admin/models/smartlink.v1.yaml

```diff
@@ -0,0 +1,112 @@
+title: Smartlink model
+type: object
+description: Smartlink schema
+x-tags:
+  - Models
+properties:
+  _id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  advs:
+    type: array
+    items:
+      type: string
+  cats:
```

Comment:
these fields will not make any sense to the customer so please alias them @Bhumika0201